src/main/java/edu/uc/cech/agrawadv/eventorganizer/user/UserService.java

```diff
@@ -5,9 +5,10 @@
 import java.util.UUID;
 
 public class UserService {
+	
     private HashMap<UUID, User> users = new HashMap<>();
 
-    User findUser(UUID userId) {
+    User findUserById(UUID userId) {
```

Comment:
The reason I changes this to findUserByID was because the other variable was named find by Username. It is important to be consistent when creating variable names. ref: https://dzone.com/articles/keep-your-code-consistent